docs/paper/dev/getting-started/userdev.mdx

```diff
@@ -24,6 +24,12 @@ This can make it hard to work with in a development environment. This plugin let
 types, names, and fields during development, and then remaps your plugin, so it can still be used with the obfuscated
 server.
 
+:::caution[1.20.5]
+
+As of Minecraft version 1.20.5, Paper ships with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings as we have traditionally done since Paper 1.17.
```

Comment:
I'm not sure the since 1.17 part is relevant. userdev users don't need to know or care about Paper's internal development process. Might add more confusion without any real benefit.